cms/templates/js/add-xblock-component-menu-problem.underscore

```diff
@@ -4,9 +4,6 @@
             <a class="link-tab" href="#tab1"><%= gettext("Common Problem Types") %></a>
         </li>
         <li>
-            <a class="link-tab" href="#tab2"><%= gettext("Common Problems with Hints and Feedback") %></a>
```

Comment:
I thought this tab was dynamically inferred from the yaml template... I'm surprised that we also statically declare it here.